x-pack/test/fleet_api_integration/apis/policy_secrets.ts

```diff
@@ -41,37 +41,43 @@ function createdPolicyToUpdatePolicy(policy: any) {
   return updatedPolicy;
 }
 
+const SECRETS_INDEX_NAME = '.fleet-secrets';
 export default function (providerContext: FtrProviderContext) {
-  // FAILING ES PROMOTION: https://github.com/elastic/kibana/issues/162732
-  describe.skip('fleet policy secrets', () => {
+  describe('fleet policy secrets', () => {
     const { getService } = providerContext;
 
     const es: Client = getService('es');
     const supertest = getService('supertest');
     const kibanaServer = getService('kibanaServer');
 
-    const getPackagePolicyById = async (id: string) => {
-      const { body } = await supertest.get(`/api/fleet/package_policies/${id}`);
-      return body.item;
+    const getSecrets = async (ids?: string[]) => {
+      const query = ids ? { terms: { _id: ids } } : { match_all: {} };
+      return es.search({
```

Comment:
I don't think you can search the index directly, does this test work locally?

Reply:
Good point. I will double check locally, but the build was green even after cristinas PR was merged I believe 🤔